cms/djangoapps/course_to_library_import/views/v0/views.py

```diff
@@ -52,3 +67,185 @@ def post(self, request, *args, **kwargs):
             override=data.validated_data['override'],
         )
         return Response({'status': 'success'})
+
+
+class CreateCourseToLibraryImportView(CreateAPIView):
+    """
+    **Use Case**
+        Allows to create course to library import.
+    **Example Request**
+        POST /api/course_to_library_import/v0/create_import/<content-library-id>/
+        **POST Parameters**
+            * course_ids (list) - A list of course IDs whose content will be saved
+            in Staged Content for further import.
+    **POST Response Values**
+        If the request is successful, an HTTP 201 "Created" response
+        is returned with the newly created CourseToLibraryImport details.
+        The HTTP 201 response has the following values.
+        {
+          "course_ids": ["course-v1:edX+DemoX+Demo_Course", "course-v1:edX+DemoX+Demo_Course2"],
+          "status": "pending",
+          "library_key": "lib:edX:1",
+          "uuid": "89b71d29-2135-4cf2-991d-e4e13b5a959a"
+        }
+    """
+
+    serializer_class = CourseToLibraryImportSerializer
+    permission_classes = (IsAdminUser,)
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+
+    def get_serializer_context(self) -> dict:
+        """
+        Extra context provided to the serializer class with current provider. We need the provider to
+        remove idp_slug from the remote_id if there is any
+        """
+        context = super().get_serializer_context()
+        context['library_id'] = self.kwargs['lib_key_str']
```

Comment:
How do we validate the mandatory keys?

Reply:
This is a parameter from the URL. 
Renamed to be consistent with the docstring.